### DISCUSSION TITLE
Updating Terminal (With Windows Installer)

### DISCUSSION THREAD

**Discussion by CaptainTollBooth** [OpenBB-finance/OpenBB#1659] [Q&A]
When running the update command I get the following: 

![image](https://user-images.githubusercontent.com/53947682/162455964-b85af5cd-649a-4a82-93b1-da225625b770.png)

Is the best way to update it to simply download the latest file and re-install?

Thanks

**Comment by DidierRLopes** [comment]
Hey @CaptainTollBooth ,

That is correct yes. 

That update command is for people that install through python (we should have made that more clear, sorry about that). 

To update from installer the best way is downloading latest file and re-install as you mentioned.

Hope this answers your question,
Didier
